src/dynarec/rv64/dynarec_rv64_66.c

```diff
@@ -718,6 +718,21 @@ uintptr_t dynarec64_66(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                 SMWRITE2();
             }
             break;
+        case 0x8E:
+            INST_NAME("MOV Seg, Ew");
+            nextop = F8;
+            u8 = (nextop & 0x38) >> 3;
+            if ((nextop & 0xC0) == 0xC0) {
```

Comment:
isn't there a MODREG  macro for that?

Reply:
Please merge this PR, I will do a global replacement in another PR, there are many places need to be replaced across all 3 backends.

Reply:
ok